akvo/rsr/static/scripts-src/rsr.js

```diff
@@ -38,6 +38,7 @@ $(document).ready(function() {
     e.preventDefault();
     $("#wrapper, #search").toggleClass("toggled");
     $("a.showFilters > i").toggleClass("fa-toggle-off fa-toggle-on");
+    Cookies.set(advancedFilterCookie, $("#wrapper, #search").hasClass('toggled')?'on':'off');
```

Comment:
Sorry for the broken PR. I think I had a local change, which I forgot to commit or something.